bin/run-tests-in-docker.sh

```diff
@@ -20,7 +20,6 @@ docker build --rm -t exercism/go-representer .
 docker run \
     --rm \
     --network none \
-    --read-only \
```

Comment:
With this being present, the result files cannot be written. The Go test runner also does not have this line.